### DISCUSSION TITLE
Specify DNS via DHCP instead of forwarding through OMR?

### DISCUSSION THREAD

**Discussion by iambenmitchell** [Ysurac/openmptcprouter#2630] [Q&A] [answered]
Hi all,

Currently devices on the network are instructed to use the router IP as the DNS server (`192.168.100.1`). However, I would like to setup my pi-hole again and the server will run on `192.168.100.6`.

How do I change the DHCP server settings to advertise `192.168.100.6` as the DNS server? My understanding is that if I change the DNS settings under OMR > Network > DHCP and DNS > DNS forwarding then clients will still have `192.168.100.1` as the DNS server but the requests are forwarded to pi-hole.

This is NOT what I want. I want my DHCP clients to receive `192.168.100.6` as their DNS server.

Any ideas? 
<img width="451" alt="image" src="https://user-images.githubusercontent.com/27426792/198845387-304c224b-217b-42d5-a1f8-74b9493c6dc3.png">

**Comment by hairy-dog** [comment]
I simply put the IP address of the local DNS servers in so mine says this:
/lan/
/use-application-dns.net/
192.168.0.19
192.168.0.24

It works for me. However, I think it might be better to run the pihole on the VPS server.

**Comment by iambenmitchell** [comment, marked as answer]
I found out how to do it.

Login to the UI, go to Network > Interfaces

Click **Edit** on the LAN network, go to DHCP server and then in DHCP Options enter in: `6,` followed by your DNS servers. In this example I am using `1.1.1.1` and `1.0.0.1`

So I enter in: `6,1.1.1.1,1.0.0.1`
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/27426792/200190132-3ba35c81-6fda-4e98-8a2b-e356ddd958a6.png">
